careers/blocks/slider/slider.js

```diff
@@ -0,0 +1,218 @@
+import { buildEllipsis } from '../../scripts/scripts.js';
+
+const DEFAULT_CONFIG = Object.freeze({
+  'visible-slides': 3,
+  maxlines: 3,
+  ellipsis: '...more',
+});
+
+const isADesktop = () => {
+  const mediaDesktop = window.matchMedia('only screen and (min-width: 769px)');
+  return mediaDesktop.matches;
+};
+
+const getPositionX = (event) => (event.type.includes('mouse')
+  ? event.pageX
+  : event.touches[0].clientX);
+
+const setSliderPosition = (currentTranslate, slider) => {
+  slider.style.transform = `translateX(${currentTranslate}px)`;
+};
+
+export default function decorate(block) {
+  const cardWrapper = document.createElement('div');
+  cardWrapper.classList.add('card-wrapper');
+
+  block.querySelectorAll('div.slider > div').forEach((div) => {
+    cardWrapper.appendChild(div);
+    div.classList.add('card');
+
+    const picture = div.querySelector('picture');
+    if (picture) {
+      const imageParent = picture.closest('div');
+      imageParent.classList.add('card-image');
+      div.classList.add('tall-card');
+    }
+
+    const contentDivs = div.querySelectorAll(':scope > div:not(.card-image)');
+    contentDivs[0].classList.add('short-description');
+  });
+
+  block.appendChild(cardWrapper);
+
+  // add slider arrow buttons
+  const slides = [...block.querySelectorAll('.card')];
+  if (slides.length > DEFAULT_CONFIG['visible-slides']) {
+    const arrowLeft = document.createElement('div');
+    arrowLeft.classList.add('slider-button', 'left');
+    block.appendChild(arrowLeft);
+
+    const arrowRight = document.createElement('div');
+    arrowRight.classList.add('slider-button', 'right');
+    block.appendChild(arrowRight);
+  }
+
+  const shortDescriptionDivs = block.querySelectorAll('.short-description');
+  shortDescriptionDivs.forEach((div) => {
+    const ellipsableText = div.querySelector('p');
```

Comment:
A lot of code here could be removed if we go full-CSS and make the text clickable.. wdyt?

Reply:
Done!